platform/admin/ui/src/main/webapp/js/views/configuration/ConfigurationEdit.view.js

```diff
@@ -255,9 +257,29 @@ define([
         /**
          * Submit to the backend.
          */
-        submitData: function() {
+        submitData: function () {
+            var spinner = new Spinner(define({
```

Comment:
Are you still going to use the default values and then extend them with your changes?